### PR TITLE
chore: adjust versions and yml files and fix mongo express authentication

### LINKED ISSUE #16: version updates and ME update
ME currently requires authentication -> disable by default

Update versions, e.g. QL, etc. and add them to env files

### PR DESCRIPTION
**Pull request by SBlechmann**

### CODE REVIEW

File: v2/docker-stack.yml [hunk 1]
```diff
@@ -332,7 +332,7 @@ services:
     image: eclipse-mosquitto:${MOSQUITTO_VERSION:-2.0.15}
```
Comment (djs0109) [inline v2/docker-stack.yml]: Here is still `2.0.15`

File: v2/docker-compose.yml [hunk 2]
```diff
@@ -10,140 +36,137 @@ services:
...
   iot-agent:
     image: fiware/iotagent-json:${IOTA_JSON_VERSION:-1.19.0}
     hostname: iot-agent
     container_name: "iot-agent"
     restart: always
-    depends_on:
-      - mongo-db
     ports:
-      - "4041:4041"
+      - "${IOTA_JSON_PORT:-4042}:${IOTA_JSON_NORTH_PORT:-4041}"
```
Comment (djs0109) [inline v2/docker-compose.yml]: Port 4041?

File: v2/.env [hunk 3]
```diff
@@ -122,29 +144,37 @@ QL_REDIS_PORT=6379
...
-MOSQUITTO_VERSION=2.0.15
+MOSQUITTO_VERSION=2.0.11 # There seems to be a bug in >2.0.11 regarding MQTT notifications, some keepalive option causes trouble.
+# External port Mosquitto shall be reached at
+MOSQUITTO_PORT=1883
+# MOSQUTTO_CONFIG_NAME=mosquitto.conf
 
 # IOT-AGENT-JSON
 IOTA_JSON_VERSION=1.26.0
+# External port IoT Agent JSON shall be reached at
+IOTA_JSON_PORT=4042
```
Comment (djs0109) [inline v2/.env]: 4041?

File: v2/docker-compose.yml [hunk 4]
```diff
@@ -1,7 +1,33 @@
 version: "3.8"
 
 services:
-  
+          
+  orion:
```
Comment (djs0109) [inline v2/docker-compose.yml:5]: I received this warning, and the Orion is therefore not reachable 
`WARN@2024-07-30T08:16:43.378Z  mongoConnectionPool.cpp[306]: mongoc driver: <mongoc> <Last server removed from topology>`

**Review by djs0109 — CHANGES_REQUESTED** [review]

File: v2/docker-compose.yml (hunk 4 above)
Comment (SBlechmann) [reply v2/docker-compose.yml:5]: seems like I forgot to comment the replica set for the standalone version

File: v2/.env [hunk 5]
```diff
@@ -153,23 +183,35 @@ IOTA_JSON_MONGODB_PORT=27017
 IOTA_JSON_MONGO_REPLICASET=rs0
 # The name of the database used in mongoDB
 IOTA_JSON_MONGODB_DB_NAME=iotagentjson
+# The provider URL used for registrations in Orion
+IOTA_JSON_PROVIDER_URL=http://iot-agent-json:4041
```
Comment (djs0109) [inline v2/.env:190]: It must be `http://iot-agent:4041`

**Review by djs0109 — CHANGES_REQUESTED** [review]

**Comment by SBlechmann** [comment]
@djs0109 I changed some more information. Please verify all functions are working as expected.

**Review by djs0109 — APPROVED** [review]

**Pull request merged**